api/api/spec/spec.yaml

```diff
@@ -12036,6 +12062,8 @@ paths:
           $ref: '#/components/responses/WrongContentTypeResponse'
         '413':
           $ref: '#/components/responses/RequestTooLargeResponse'
+        '415':
+          $ref: '#/components/responses/UnsupportedMediaTypeResponse'
```

Comment:
I would suggest to validate this in another issue. 

The `Body.validate_content_type` class method raises a `WazuhNotAcceptable(6002)`. This exception is now handled by the generic error handler `problem_error_handler`. I think that it will be better to create a custom exception inherited from `ProblemException` to generate the same response and notify the content_type error.

Reply:
This request generates a 415 error response. The header `Content-Type` does not meet the defined type in the `spec.yaml`, which should be `application/octet-stream`. 

```console
curl -k -X PUT "https://localhost:55000/manager/configuration?wait_for_complete=true&pretty=true" -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"
{
   "title": "Unsupported Media Type",
   "detail": "Invalid Content-type (application/json), expected ['application/octet-stream']"
}
```
Every endpoint with the `content-type` defined in the spec will generate a 415 error if the request defines another `content-type`.

Reply:
I tried again and it returns 415 as you said. Maybe there was something wrong with my mappings.
![image](https://github.com/wazuh/wazuh/assets/23361101/31b8d1da-a57d-42f8-98cd-f357f7825f82)